owncloudApp/src/main/res/layout/log_item.xml

```diff
@@ -16,24 +15,10 @@
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              android:layout_width="wrap_content"
-              android:layout_height="wrap_content"
-              android:layout_marginHorizontal="@dimen/standard_margin"
-              android:layout_marginVertical="@dimen/standard_half_margin"
-              android:filterTouchesWhenObscured="true"
-              android:orientation="vertical">
-
-    <TextView
-        android:id="@+id/logTimestamp"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="adios"/>
-
-    <TextView
-        android:id="@+id/logContent"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="hola"/>
-
-</LinearLayout>
\ No newline at end of file
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/logContent"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:lines="1"
+    tools:text="hola" />
```

Comment:
Please delete this line and `xmlns:tools="http://schemas.android.com/tools"` since are not needed.

Reply:
It's used !
`tools:text="hola"`